Updates vignette (#148)
NorskRegnesentral/shapr: vignettes/understanding_shapr.Rmd

foundation (@lundberg2017unified). Kernel SHAP is a computationally efficient approximation to 
Shapley values in higher dimensions, but it assumes independent features. @aas2019explaining extend 
the Kernel SHAP method to handle dependent features, resulting in more accurate approximations to 

> the true Shapley values. [See the paper (@aas2019explaining) for further details](https://arxiv.org/abs/1903.10464).

Reviewer: ```suggestion
the true Shapley values. See the [paper]((https://arxiv.org/abs/1903.10464))(@aas2019explaining) for further details.
```

```{r}
# Fitting a gbm model using caret
set.seed(825)
model <- caret::train(
  x_train,
  y = y_train,
  method = "gbm",
  verbose = FALSE
)

# Create custom function of model_type for caret
model_type.train <- function(x) {
  ifelse(
    x$modelType[[1]] == "Classification",
    "classification",
    "regression"
  )
}

# Create custom function of predict_model for caret
predict_model.train <- function(x, newdata) {

>   if (!requireNamespace('caret', quietly = TRUE)) {

Reviewer: ```suggestion
  if (!requireNamespace("caret", quietly = TRUE)) {
```

>     stop('The caret package is required for predicting train models')

Reviewer: ```suggestion
    stop("The caret package is required for predicting train models")
```
